packages/bruno-app/src/components/Sidebar/CreateCollection/index.js

```diff
@@ -257,6 +291,18 @@ const CreateCollection = ({ onClose }) => {
             <div className="flex justify-between items-center mt-8 bruno-modal-footer">
               <div className="flex advanced-options">
                 <Dropdown onCreate={onDropdownCreate} icon={<AdvancedOptions />} placement="bottom-start">
+                  {shouldShowAccordion && (
```

Comment:
I don't see an Accordian based UI in the Create Collection component. We should be using some other term I feel.